sparrow-py/pysrc/sparrow_py/_timestream.py

```diff
@@ -952,6 +952,36 @@ def cast(self, data_type: pa.DataType) -> Timestream:
         """
         return Timestream(self._ffi_expr.cast(data_type))
 
+    def seconds_since(self, time: Union[Timestream, datetime]) -> Timestream:
```

Comment:
We could use this to implement `seconds_since_previous().` We could even have `e.seconds_since()` be the seconds since the previous one, and `e.seconds_since(f)` be the second since `f`.